modules/statics/src/base.ts

```diff
@@ -1324,6 +1324,42 @@ export enum UnderlyingAsset {
   // Cardano Token
   adaTestnetToken = 'temporary-placeholder',
 
+  // solana token
+  'SHDW-USDC' = 'SHDW-USDC',
+  'RENDOGE-USDC' = 'RENDOGE-USDC',
+  'whETH-USDC' = 'whETH-USDC',
+  'LINK-USDC' = 'LINK-USDC',
+  'USDT-USDC' = 'USDT-USDC',
+  'wbWBNB-USDC' = 'wbWBNB-USDC',
+  'MATICPO-USDC' = 'MATICPO-USDC',
+  'PRISM-USDC' = 'PRISM-USDC',
+  'SRM-USDC' = 'SRM-USDC',
+  'AVAX-USDC' = 'AVAX-USDC',
+  'FIDA-USDC' = 'FIDA-USDC',
+  'SUSHI-USDC' = 'SUSHI-USDC',
+  'FIDA-USDT' = 'FIDA-USDT',
+  'FTT-FTT' = 'FTT-FTT',
+  '3uejH-USDC' = '3uejH-USDC',
+  'SRMet-SRM' = 'SRMet-SRM',
+  'SOL-wtUST' = 'SOL-wtUST',
+  'ELU-USDT' = 'ELU-USDT',
+  'BOP-USDC' = 'BOP-USDC',
+  'LQID-USDC' = 'LQID-USDC',
+  'MSOL-SOL' = 'MSOL-SOL',
+  'wtUST-USDT' = 'wtUST-USDT',
+  'MSOL-USDC' = 'MSOL-USDC',
+  'XCOPE-USDC' = 'XCOPE-USDC',
+  'tuAtlas' = 'tuAtlas',
+  'tuSAMO' = 'tuSAMO',
+  'tuDFL' = 'tuDFL',
+  'tuLIKE' = 'tuLIKE',
+  'tuPOLIS' = 'tuPOLIS',
+  'tuSRM' = 'tuSRM',
+  'tuREAL' = 'tuREAL',
+  'tuMBS' = 'tuMBS',
+  'tuGENE' = 'tuGENE',
+  'tuCOPE' = 'tuCOPE',
```

Comment:
Could we use lowercase for all token names? these combinations seem a bit arbitrary
Other sol tokens use uppercase for the key and lowercase for the value